kubernetes/cray-sysmgmt-health/templates/prometheus/istio/istio-alerts.yaml

```diff
@@ -0,0 +1,31 @@
+{{- /*
+Copyright 2021 Hewlett Packard Enterprise Development LP
```

Comment:
This is using an old style of copyright notice. Since the switch to open source we need to use a different copyright header. It's documented in our contributing guide: https://connect.us.cray.com/confluence/display/CASMPET/CONTRIBUTING#CONTRIBUTING-Copyright

Reply:
> This is using an old style of copyright notice. Since the switch to open source we need to use a different copyright header. It's documented in our contributing guide: https://connect.us.cray.com/confluence/display/CASMPET/CONTRIBUTING#CONTRIBUTING-Copyright

updated